src/components/Table/Table.jsx

```diff
@@ -134,7 +134,7 @@ class Table extends React.Component {
       data = data.slice(currentPage * pageLength, (currentPage + 1) * pageLength);
     }
     return (
-      <StyledBody>
+      <StyledBody style={this.props.fixedStyle}>
```

Comment:
Planning to make this prop a class name instead which can be generalised for other use cases